src/adr/0051-email-rate-limiting.md

```diff
@@ -0,0 +1,40 @@
+# Email rate limiting
+
+Date: 2021-01-10
+
+## Status
+
+// todo
+
+## Context
+
+Rate limiting is one of the common features. Examples: SaaS is one https://www.fastmail.help/hc/en-us/articles/1500000277382-Account-limits#sending/receiving, https://support.google.com/mail/answer/22839
+They limit how many emails you can send/receive from/to each email account over a given period of time.  
+We believe the rate-limiting will help James has more control of the resource and easy to dynamic config the user policy. It also complements the storage quota.
+
```

Comment:
Limiting the amount of email sent to third parties can also prevent them from considering you as an open relay and can be beneficial as well.